addon/components/uni-carrousel.js

```diff
@@ -0,0 +1,54 @@
+import Ember from 'ember';
+import layout from '../templates/components/uni-carrousel';
+
+const { Component, $ } = Ember;
+
+export default Component.extend({
+  classNames: ['uni-carrousel'],
+  layout,
+  page: 0,
+  componentWidth: 0,
+  items: [],
+  markers: '.uni-carrousel__container__markers__item',
+
+  _update() {
+    this.set('deltaX', this.get('page') * this.get('componentWidth') * -1);
+
+    if (this.get('page') === 0) {
```

Comment:
couldnt this be a computed?
```js
showPrev: Ember.computed.eq('page', 0)
```

same for showNext. Then on the template, just wrap the buttons